### PR TITLE
shalinee | test | added tooltip on company branch name of sidebar

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
                 <div class="company-list mt-2">
                     <ul class="list-style-none font-14"
                         *ngIf="companyListForFilter && companyListForFilter.length && !(isCompanyRefreshInProcess$ | async)">
-                        <li class="btn-group" *ngFor="let company of companyListForFilter;" [ngClass]="{'active': company.uniqueName === activeCompany?.uniqueName}">
+                        <li class="btn-group" *ngFor="let company of companyListForFilter;" [ngClass]="{'active': company.uniqueName === activeCompany?.uniqueName}" tooltip="{{company.name}}" placement="top">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html:40]:
```suggestion
                        <li class="btn-group" *ngFor="let company of companyListForFilter;" [ngClass]="{'active': company.uniqueName === activeCompany?.uniqueName}" [tooltip]="company.name" placement="top">
```

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html [hunk 2]
```diff
@@ -66,7 +66,7 @@
                 <div class="company-list branch-list mt-2">
                     <ul class="list-style-none font-14" *ngIf="!branchRefreshInProcess">
                         <ng-container *ngIf="companyBranches?.branchCount > 1">
-                            <li *ngIf="activeCompany?.uniqueName !== companyBranches?.uniqueName || currentBranchUniqueName">
+                            <li *ngIf="activeCompany?.uniqueName !== companyBranches?.uniqueName || currentBranchUniqueName" tooltip="{{companyBranches.name}}"  placement="top">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html:69]:
```suggestion
                            <li *ngIf="activeCompany?.uniqueName !== companyBranches?.uniqueName || currentBranchUniqueName" [tooltip]="companyBranches?.name" placement="top">
```

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html [hunk 3]
```diff
@@ -79,7 +79,8 @@
                             </li>
                             <li *ngFor="let branch of companyBranches?.branches;">
                                 <a class="d-flex align-items-center" [ngClass]="{'active': branch?.uniqueName === currentBranchUniqueName}" href="javascript:;"
-                                    (click)="changeBranch(companyBranches.uniqueName, branch?.uniqueName, $event)">
+                                    (click)="changeBranch(companyBranches.uniqueName, branch?.uniqueName, $event)"
+                                     tooltip="{{branch?.alias}}"  placement="top">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html:83]:
```suggestion
                                     [tooltip]="branch?.alias" placement="top">
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**